### PR TITLE
Fix psr2

### PR DESCRIPTION
**Pull request by loiclau**
<h1> Pull Request Template </h1>

<h2> Description </h2>

<h2> Type of change </h2>

- [x] Updating style (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [x] 19.04.x (master)

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon

### CODE REVIEW

**Comment by vhr** [comment]
If those code style correction will be applied after that will be impossible to merge/rebase the future branch **bam-393-new-cc-merge-master** https://github.com/centreon/centreon/compare/bam-393-new-cc-merge-master

File: www/include/Administration/parameters/engine/form.php [hunk 1]
```diff
@@ -133,8 +134,10 @@
 $form->registerRule('is_writable_file_if_exist', 'callback', 'is_writable_file_if_exist');
 $form->registerRule('isNum', 'callback', 'isNum');
 
-// $form->addRule('nagios_path_img', _("The directory isn't valid"), 'is_valid_path_images'); - Field is not added so no need for rule
-// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path'); - Field is not added so no need for rule
+// $form->addRule('nagios_path_img', _("The directory isn't valid"), 'is_valid_path_images');
```
Comment (sc979) [inline www/include/Administration/parameters/engine/form.php]:
```suggestion
```

File: www/include/Administration/parameters/engine/form.php [hunk 2]
```diff
@@ -133,8 +134,10 @@
...
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/engine/form.php]:
```suggestion
```

File: www/include/Administration/parameters/engine/form.php [hunk 3]
```diff
@@ -133,8 +134,10 @@
...
+// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path');
```
Comment (sc979) [inline www/include/Administration/parameters/engine/form.php]:
```suggestion
```

File: www/include/Administration/parameters/engine/form.php [hunk 4]
```diff
@@ -133,8 +134,10 @@
...
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/engine/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 5]
```diff
@@ -234,15 +237,18 @@
 $form->registerRule('is_writable_file', 'callback', 'is_writable_file');
 $form->registerRule('is_writable_file_if_exist', 'callback', 'is_writable_file_if_exist');
 $form->addRule('oreon_path', _("Can't write in directory"), 'is_valid_path');
-// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path'); - Field is not added so no need for rule
+// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path');
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 6]
```diff
@@ -234,15 +237,18 @@
...
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 7]
```diff
@@ -234,15 +237,18 @@
...
+// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path');
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 8]
```diff
@@ -234,15 +237,18 @@
...
+// - Field is not added so no need for rule
+// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path');
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 9]
```diff
@@ -234,15 +237,18 @@
...
 $form->registerRule('is_writable_file_if_exist', 'callback', 'is_writable_file_if_exist');
 $form->addRule('oreon_path', _("Can't write in directory"), 'is_valid_path');
-// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path'); - Field is not added so no need for rule
+// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path');
+// - Field is not added so no need for rule
+// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path');
+// - Field is not added so no need for rule
+// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path');
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/general/form.php [hunk 10]
```diff
@@ -234,15 +237,18 @@
 $form->registerRule('is_writable_file', 'callback', 'is_writable_file');
 $form->registerRule('is_writable_file_if_exist', 'callback', 'is_writable_file_if_exist');
 $form->addRule('oreon_path', _("Can't write in directory"), 'is_valid_path');
-// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
-// $form->addRule('nagios_path', _("The directory isn't valid"), 'is_valid_path'); - Field is not added so no need for rule
+// $form->addRule('nagios_path_plugins', _("Can't write in directory"), 'is_writable_path');
+// - Field is not added so no need for rule
+// $form->addRule('nagios_path_img', _("Can't write in directory"), 'is_writable_path');
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/general/form.php]:
```suggestion
```

File: www/include/Administration/parameters/ldap/ldap.php [hunk 11]
```diff
@@ -1,43 +1,43 @@
 <?php
 /*
- * Copyright 2005-2012 Centreon
+ * Copyright 2005-2019 Centreon
  * Centreon is developped by : Julien Mathis and Romain Le Merlus under
```
Comment (sc979) [inline www/include/Administration/parameters/ldap/ldap.php]:
```suggestion
 * Centreon is developed by : Julien Mathis and Romain Le Merlus under
```

File: www/include/Administration/parameters/rrdtool/form.php [hunk 12]
```diff
@@ -95,13 +95,14 @@
 $form->addRule('rrdcached_port', _('The port must be numeric'), 'numeric');
 
 $form->addRule('rrdtool_path_bin', _("Can't execute binary"), 'is_executable_binary');
-// $form->addRule('oreon_rrdbase_path', _("Can't write in directory"), 'is_writable_path'); - Field is not added so no need for rule
+// $form->addRule('oreon_rrdbase_path', _("Can't write in directory"), 'is_writable_path');
```
Comment (sc979) [inline www/include/Administration/parameters/rrdtool/form.php]:
```suggestion
```

File: www/include/Administration/parameters/rrdtool/form.php [hunk 13]
```diff
@@ -95,13 +95,14 @@
...
+// - Field is not added so no need for rule
```
Comment (sc979) [inline www/include/Administration/parameters/rrdtool/form.php]:
```suggestion
```

**Comment by ganoze** [comment]
Should be closed, right @loiclau ?

**Comment by ganoze** [comment]
Outdated because of PSR-12 and unmergeable.

**Pull request closed without merging**